pdc_client/__init__.py

```diff
@@ -193,9 +193,9 @@ def get_paged(self, res, **kwargs):
         if self.page_size is not None:
             kwargs['page_size'] = self.page_size
 
-        if self.page_size <= 0 and self.page_size is not None:
-            # If page_size <= 0, pagination will be disable.
-            return res(**kwargs)
+            if self.page_size <= 0:
```

Comment:
It can't be `None` as I also moved it into the `if self.page_size is not None:` section, see the added white-spaces.